server/engine/src/main/java/org/eclipse/lsp/cobol/implicitDialects/cics/utility/CICSOptionsCheckUtility.java

```diff
@@ -26,6 +27,12 @@
 public class CICSOptionsCheckUtility {
   private final Map<Integer, CICSOptionsCheckBaseUtility> optionsMap = new HashMap<>();
 
+  private final Map<Integer, CICSOptionsCheckBaseUtility> spOptionsMap = new HashMap<>();
+
+  @Setter private static boolean exciOptionsEnabled = false;
+
+  @Setter private static boolean spOptionsEnabled = true;
```

Comment:
Is this the correct default? It look like both the compiler and preprocessor default to NOSP.